### PR TITLE
Fixes for osc-1.x

### PR DESCRIPTION
**Pull request by dirkmueller**
Starting with the minimal changes from #2879 again

### CODE REVIEW

File: check_tags_in_requests.py [hunk 1]
```diff
@@ -72,7 +74,7 @@ def _check_requests(self, project, package, rev):
                         project = project[len(prefix):]
                         sr = srprefix
                         break
-            requests = osc.core.get_request_list(apiurl, project, package, None, ['new', 'review'], 'submit')
+            requests = get_request_list_with_history(apiurl, project, package, None, ['new', 'review'], 'submit')
```
Comment (Vogtinator) [inline check_tags_in_requests.py:77]: This didn't use `withfullhistory`, so it should behave the same with the compatibility wrapper. Were you able to find the cause of the difference?

File: osclib/core.py [hunk 2]
```diff
@@ -56,10 +56,52 @@ def groups_members(apiurl, groups):
 
     return members
 
-# osc uses xml.etree while we rely on lxml
+
+def get_request_list_with_history(
```
Comment (Vogtinator) [inline osclib/core.py:60]: Can you add a deprecation warning or at least a `# TODO` comment?

File: check_tags_in_requests.py (hunk 1 above)
Comment (dirkmueller) [reply check_tags_in_requests.py:77]: Yes, the return values are different. I can search the details tomorrow. for now I'd like to get this in to have a green CI to iterate on improving the quality on this.

File: osclib/core.py (hunk 2 above)
Comment (dirkmueller) [reply osclib/core.py:60]: there's one on the line below (63). I can make it a deprecation warning when everything else is green.

Comment (Vogtinator) [reply osclib/core.py:60]: AFAIK a warning wouldn't make it less green than it was previously, wouldn't it?

Comment (dirkmueller) [reply osclib/core.py:60]: It does not. just want to get to see that everything works before adding further code changes (that would restart the CI). do want to leave from work today at some point soon ;)

Comment (Vogtinator) [reply osclib/core.py:60]: Ah, so it's just about the push, I see. In that case we'll need a separate PR anyway.

> do want to leave from work today at some point soon ;)

Protip: You can do that while the CI is running - /me writes this reply from home :stuck_out_tongue:

**Pull request merged**